app/src/renderer/js/export.js

```diff
@@ -0,0 +1,55 @@
+import {ipcRenderer, remote} from 'electron';
+import {handleTrafficLightsClicks} from '../js/utils';
+
+const {dialog} = remote;
+
+let isExportInProgress = false;
+
+window.onbeforeunload = e => {
+  if (isExportInProgress) {
+    const buttonIndex = dialog.showMessageBox(remote.getCurrentWindow(), {
+      type: 'question',
+      buttons: ['Cancel Export', 'Continue'],
+      defaultId: 1,
+      message: 'Are you sure you want to cancel exporting?',
+      detail: 'It will not be saved'
```

Comment:
Needs a `.` at the end.

Exporting is not just about saving, a plugin might upload it instead of saving, so this needs a better wording. Or is it needed at all? I think it's pretty clear already in the `message` what's going to happen.

Reply:
@sindresorhus I don't think this is needed. I copied this dialog from the editor window.